app/src/comic/serializers.py

```
@@ -0,0 +1,119 @@
+
+
+class UserSerializer(serializers.ModelSerializer):
+    """
+    用戶序列化器
+    """
+
+    class Meta:
+        from django.contrib.auth.models import User
+
+        model = User
+        fields = ["id", "username", "email", "date_joined"]
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)